packages/api/src/models/sensorReadingModel.js

```diff
@@ -87,6 +87,7 @@ class SensorReading extends Model {
     return await SensorReading.query()
       .select('*')
       .joinRaw('JOIN sensor ON sensor_reading.location_id::uuid = sensor.location_id')
+      .joinRaw('JOIN location ON location.location_id::uuid = sensor.location_id::uuid')
```

Comment:
As far as I know, all of the columns for location_id are already type `uuid`, so I believe we do not need to cast them to type `uuid`

Reply:
Done